contrib/sip_proxy/filters/network/source/conn_manager.cc

```diff
@@ -545,14 +545,12 @@ ConnectionManager::ActiveTrans::upstreamData(MessageMetadataSharedPtr metadata)
     return SipFilters::ResponseStatus::MoreData;
   } catch (const AppException& ex) {
     ENVOY_LOG(error, "sip response application error: {}", ex.what());
-    // parent_.stats_.response_decoding_error_.inc();
```

Comment:
revert this and below.

Reply:
These two lines are also comment out and invalid in previous version. I just remove the comment out lines.